sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -61,7 +60,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultScheduler.class);
 
-    protected final ExecutorService executor = Executors.newFixedThreadPool(1);
+    /**
+     * 1 Thread for processing offers off the queue in {@link #processOffers()}
```

Comment:
The `processOffers` call never releases its thread and the ordering of Scheduler startup precludes this possibility right now, but we could have separate executor services to avoid future developers hitting this accidentally.